climpred/bootstrap.py

```diff
@@ -405,20 +472,26 @@ def bootstrap_perfect_model(
     control,
     metric='pearson_r',
     comparison='m2e',
+    dim=None,
     sig=95,
     bootstrap=500,
     pers_sig=None,
+    **metric_kwargs,
 ):
     """Wrapper for bootstrap_compute for perfect-model in steady state."""
```

Comment:
Same as in `bootstrap_hindcast`. Would be good to just copy and paste the docstring from `bootstrap_compute`